support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -1165,7 +1185,7 @@ function CheckoutComponent({ geoId }: Props) {
 															]}
 														>
 															<Radio
-																label={label}
+																label={getLabelText(label, selected)}
```

Comment:
Could we just have this in s ternary here to keep the code a little more succinct?

Reply:
Yep just doen that in the ocmment above